tsconfig.json

```diff
@@ -13,19 +21,16 @@
     "experimentalDecorators": true,
     "allowSyntheticDefaultImports": true,
     "noUnusedParameters": true,
-    "target": "es2015",
-    "typeRoots": [
-      "node_modules/@types"
-    ],
-    "lib": [
-      "es2016",
-      "dom"
-    ]
+    "target": "es2017",
+    "types": ["node"],
+    "typeRoots": ["node_modules/@types"],
+    "lib": ["es2020", "dom"]
   },
-  "exclude": [
-    "*.eot",
-    "*.woff2",
-    "*.svg",
-    "*.woff"
-  ]
+  "exclude": ["*.eot", "*.woff2", "*.svg", "*.woff"]
```

Comment:
are these for font-awesome files? I still see all those in dist artificats. I wonder if we need specific